sample-java-client/README.md

```diff
@@ -0,0 +1,170 @@
+*MtoServiceItemApi* | [**createMTOServiceItem**](docs/MtoServiceItemApi.md#createMTOServiceItem) | **POST** /move-task-orders/{moveTaskOrderID}/mto-shipments/{mtoShipmentID}/mto-service-items | Creates MTO service items that is added to a Move Task Order and MTO Shipment
+*MtoShipmentApi* | [**updateMTOShipment**](docs/MtoShipmentApi.md#updateMTOShipment) | **PUT** /move-task-orders/{moveTaskOrderID}/mto-shipments/{mtoShipmentID} | Updates MTO shipment
+*PaymentRequestsApi* | [**createPaymentRequest**](docs/PaymentRequestsApi.md#createPaymentRequest) | **POST** /payment-requests | Creates a payment request
+*UploadsApi* | [**createUpload**](docs/UploadsApi.md#createUpload) | **POST** /payment-requests/{paymentRequestID}/uploads | Create a new upload for a payment request
+
+
+## Documentation for Models
+
+ - [Address](docs/Address.md)
+ - [Body](docs/Body.md)
+ - [ClientError](docs/ClientError.md)
+ - [CreatePaymentRequestPayload](docs/CreatePaymentRequestPayload.md)
+ - [Customer](docs/Customer.md)
+ - [CustomerContactType](docs/CustomerContactType.md)
+ - [DimensionType](docs/DimensionType.md)
+ - [DutyStation](docs/DutyStation.md)
+ - [Entitlements](docs/Entitlements.md)
+ - [Error](docs/Error.md)
+ - [MTOAgent](docs/MTOAgent.md)
+ - [MTOAgentType](docs/MTOAgentType.md)
+ - [MTOAgents](docs/MTOAgents.md)
+ - [MTOServiceItem](docs/MTOServiceItem.md)
+ - [MTOServiceItemBasic](docs/MTOServiceItemBasic.md)
+ - [MTOServiceItemDDFSIT](docs/MTOServiceItemDDFSIT.md)
+ - [MTOServiceItemDOFSIT](docs/MTOServiceItemDOFSIT.md)
+ - [MTOServiceItemDimension](docs/MTOServiceItemDimension.md)
+ - [MTOServiceItemDomesticCrating](docs/MTOServiceItemDomesticCrating.md)
+ - [MTOServiceItemModelType](docs/MTOServiceItemModelType.md)
+ - [MTOServiceItemShuttle](docs/MTOServiceItemShuttle.md)
+ - [MTOServiceItemStatus](docs/MTOServiceItemStatus.md)
+ - [MTOShipment](docs/MTOShipment.md)
+ - [MTOShipmentType](docs/MTOShipmentType.md)
+ - [MTOShipments](docs/MTOShipments.md)
+ - [MoveOrder](docs/MoveOrder.md)
+ - [MoveTaskOrder](docs/MoveTaskOrder.md)
+ - [MoveTaskOrders](docs/MoveTaskOrders.md)
+ - [PaymentRequest](docs/PaymentRequest.md)
+ - [PaymentRequestStatus](docs/PaymentRequestStatus.md)
+ - [PaymentRequests](docs/PaymentRequests.md)
+ - [ProofOfServiceDocs](docs/ProofOfServiceDocs.md)
+ - [ReServiceCode](docs/ReServiceCode.md)
+ - [ServiceItem](docs/ServiceItem.md)
+ - [ServiceItemParams](docs/ServiceItemParams.md)
+ - [UpdatePaymentRequestStatus](docs/UpdatePaymentRequestStatus.md)
+ - [Upload](docs/Upload.md)
+ - [ValidationError](docs/ValidationError.md)
+
+
+## Documentation for Authorization
+
+All endpoints do not require authorization.
+Authentication schemes defined for the API:
+
+## Recommendation
+
+It's recommended to create an instance of `ApiClient` per thread in a multithreaded environment to avoid any potential issues.
+
+## Author
+
+dp3@truss.works
+
```

Comment:
Has anyone tried out this client recently? Should we?

Reply:
I'm not really sure how to try it out

Reply:
I can try to walk through this README, see if it actually works!